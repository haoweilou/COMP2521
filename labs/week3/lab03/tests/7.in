20 10 5 3 7 30 25 35 23 27 4 6 22 28
