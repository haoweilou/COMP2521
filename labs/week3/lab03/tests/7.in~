20 10 5 3 7 30 25 23 27 4 6 22 28
