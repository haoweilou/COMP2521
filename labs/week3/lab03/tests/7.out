Tree:
           20
           / \
          /   \
         /     \
        /       \
       /         \
      10         30
     /           / \
    5           /   \
   / \         25   35
  /   \       / \
 /     \     /   \
3       7   23   27
 \     /   /       \
  4   6   22       28

#nodes = 14,  #leaves = 5
Infix  : 3 4 5 6 7 10 20 22 23 25 27 28 30 35 
Prefix : 20 10 5 3 4 7 6 30 25 23 22 27 28 35 
Postfix: 4 3 6 7 5 10 22 23 28 27 25 35 30 20 
ByLevel: 20 10 30 5 25 35 3 7 23 27 4 6 22 28 
